commercetools-sdk-base/src/main/java/io/sphere/sdk/search/model/TermFilterExistsAndMissingSearchModel.java

```diff
@@ -0,0 +1,44 @@
+package io.sphere.sdk.search.model;
+
+import io.sphere.sdk.search.FilterExpression;
+
+import java.util.List;
+import java.util.function.Function;
+
+/**
+ * Model to build term filters with exits and missing predicates.
+ * .
+ * @param <T> type of the resource
+ * @param <V> type of the value
+ */
+public interface TermFilterExistsAndMissingSearchModel<T, V> extends TermFilterSearchModel<T, V>, ExistsAndMissingFilterSearchModelSupport<T> {
+    @Override
+    List<FilterExpression<T>> containsAll(final Iterable<V> values);
+
+    @Override
+    List<FilterExpression<T>> containsAllAsString(final Iterable<String> values);
+
+    @Override
+    List<FilterExpression<T>> containsAny(final Iterable<V> values);
+
+    @Override
+    List<FilterExpression<T>> containsAnyAsString(final Iterable<String> values);
+
+    @Override
+    List<FilterExpression<T>> is(final V value);
+
+    @Override
+    List<FilterExpression<T>> isIn(final Iterable<V> values);
```

Comment:
I wasn't sure why we repeat overridden methods when we extend some interfaces and tried to follow our code style. The only reason I had on my mind was that we did it so that javadoc copies the doc from the extended interface. But it does it automatically for non-public interfaces and for public interfaces you can jump to the javadoc of the super interfaces. 

Then I would suggest to remove these overriden methods when working on such an interface again 👍